recipes/compas/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "COMPAS" %}
+{% set version = "0.3.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "16b825123798359c3ed1a08afbf426ba20b1271c01d9561bea260c1dc698ce1a" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - matplotlib
+    - networkx
+    - numba
+    - numpy
+    - scipy
+    - sympy
+  run:
+    - python
+    - matplotlib
+    - networkx
+    - numba
+    - numpy
+    - scipy
+    - sympy
+
+test:
+  imports:
+    - compas
+    - compas_blender
+    - compas_ghpython
+    - compas_rhino
+
+about:
+  home: http://compas-dev.github.io
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: The COMPAS framework
+  description: |
+    COMPAS is an open-source, Python-based computational framework
+    for collaboration and research in architecture, engineering and
+    digital fabrication. It is developed by the Block Research
+    Group (https://block.arch.ethz.ch) of ETH Zurich (http://www.ethz.ch),
+    with the support of the National Centre for Competence in
+    Research on "Digital Fabrication" (http://www.dfab.ch).
+  doc_url: https://compas-dev.github.io
+  dev_url: https://github.com/compas-dev/compas
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    - brgcode
```

Comment:
Has the listed co-maintainer signed off on being listed? If not, please ping them here and get permission.

Reply:
yes, @brgcode is the main developer of compas, and we work together in the same institute.